registry/remote/repository.go

```diff
@@ -1070,15 +1086,15 @@ func verifyContentDigest(resp *http.Response, expected digest.Digest) error {
 	contentDigest, err := digest.Parse(digestStr)
 	if err != nil {
 		return fmt.Errorf(
-			"%s %q: invalid response header: `%s: %s`",
+			"%s %q: invalid response header `%s=%q`",
```

Comment:
Let's revert this change since `%s: %s` is the original form in the HTTP protocol.
```suggestion
			"%s %q: invalid response header: `%s: %s`",
```